cmd/axelard/cmd/vald/start.go

```diff
@@ -320,30 +323,47 @@ func createJob(sub tmEvents.FilteredSubscriber, processor func(event tmEvents.Ev
 
 }
 
-func getStartBlock(stateStore StateStore, tmClient tmEvents.BlockHeightClient, logger log.Logger) (int64, error) {
+// Get the block height to start listening for TM events from.
+// Checks that the node is not too out of sync from the network.
+// If state.json is not stale, uses that as starting height.
+// Otherwise, starts from the node height.
+func getStartBlock(cfg config.ValdConfig, stateStore StateStore, tmClient tmEvents.BlockHeightClient, logger log.Logger) (int64, error) {
 	startBlock, err := stateStore.GetState()
 	if err != nil {
-		logger.Error(err.Error())
-
-		return 0, nil
+		logger.Info(err.Error())
```

Comment:
Please move `stateStore.GetState()` for `startBlock` closer to where `startBlock` is used so that we see error checks first for this function and then see the logic related to `startBlock`

Reply:
Makes sense